app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -35,6 +38,11 @@ fun StatinNudge(
 ) {
   AnimatedVisibility(
       visible = isVisible,
+      enter = expandVertically(
```

Comment:
Rather than expand, shouldn't it be sliding in and out? Atleast that's what I remember from the CDSS nudge animation 🤔